bin/bst.ts

```diff
@@ -60,4 +61,14 @@ if (command === "help") {
     console.log("bst debug <agent-id> <service-port>        Forwards traffic from Alexa to your local Skill service, listening on <service-port>");
     console.log("bst debug-url <agent-id> <alexa-url>       Takes a normal URL and modifies to include the <agent-id> in the query string");
     console.log("");
-}
\ No newline at end of file
+}
+
+if (command === "lambda-package") {
```

Comment:
This should be "package lambda" so it is consistent with proxy command (such as "proxy lambda")